### ISSUE TITLE
ConnectionSettings.hostIp returns null

### ISSUE THREAD

**Issue by Youssef-Ait-Elhaj** [epam/sap-commerce-intellij-idea-plugin#1094]
getting this exception when starting intelliJ

```
java.lang.NullPointerException: connectionSettings.hostIP must not be null
	at com.intellij.idea.plugin.hybris.common.services.impl.DefaultCommonIdeaService.cleanUpRemoteConnectionSettingsHostIp(DefaultCommonIdeaService.kt:218)
	at com.intellij.idea.plugin.hybris.common.services.impl.DefaultCommonIdeaService.prepareSslRemoteConnectionSettings(DefaultCommonIdeaService.kt:213)
	at com.intellij.idea.plugin.hybris.common.services.impl.DefaultCommonIdeaService.fixRemoteConnectionSettings$lambda-7(DefaultCommonIdeaService.kt:190)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at com.intellij.idea.plugin.hybris.common.services.impl.DefaultCommonIdeaService.fixRemoteConnectionSettings(DefaultCommonIdeaService.kt:186)
	at com.intellij.idea.plugin.hybris.project.components.HybrisProjectManagerListener.continueOpening(HybrisProjectManagerListener.java:128)
	at com.intellij.idea.plugin.hybris.project.components.HybrisProjectManagerListener.lambda$projectOpened$1(HybrisProjectManagerListener.java:109)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivities(StartupManagerImpl.kt:394)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivities$default(StartupManagerImpl.kt:370)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivitiesRegisteredDynamically(StartupManagerImpl.kt:327)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:275)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:70)
	at com.intellij.ide.startup.impl.StartupManagerImpl$projectOpened$$inlined$forkJoinTask$1$lambda$1.invoke(StartupManagerImpl.kt:189)
	at com.intellij.ide.startup.impl.StartupManagerImpl$projectOpened$$inlined$forkJoinTask$1$lambda$1.invoke(StartupManagerImpl.kt:70)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$sam$java_lang_Runnable$0.run(StartupManagerImpl.kt)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:365)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:343)
	at com.intellij.ide.startup.impl.StartupManagerImpl$projectOpened$$inlined$forkJoinTask$1.run(trace.kt:90)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1375)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

**Comment by mlytvyn** [comment]
@frostyy9 , thank you for reporting this issue, I’ll fix it in the upcoming release. Stay tuned!

**Comment by mlytvyn** [comment]
@frostyy9 , can you please provide IDEA and Plugin details?

p.s. upgrade to latest version of the Plugin.

**Comment by Youssef-Ait-Elhaj** [comment]
The problem can be reproduced with the following versions:
IntelliJ IDEA version 2024 with latest version of SAP Commerce Developers Toolset

**Comment by Youssef-Ait-Elhaj** [comment]
> @frostyy9 , can you please provide IDEA and Plugin details?
> 
> p.s. upgrade to latest version of the Plugin.

Already done that, but issue persisted, had to downgrade to IntelliJ version 2022.3 for it work

**Comment by mlytvyn** [comment]
@frostyy9 , there is no such method in the Plugin 2024.1.0 https://plugins.jetbrains.com/plugin/12867-sap-commerce-developers-toolset/versions/stable/491133
That method was removed here https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/971.

Can you please share exception when using latest version of the IDEA and Plugin 2024.1.0

**Comment by Youssef-Ait-Elhaj** [comment]
@mlytvyn Maybe the issue was on my part and I didn't actually upgrade SCT version, and that's why it crashed, because I used IntelliJ for the update so it might not have done a good job (should've done it manually)

**Comment by mlytvyn** [comment]
Is it resolved now?

**Comment by mlytvyn** [comment]
@frostyy9 , should be fixed in `2024.1.1`
